lib/pa11y.js

```diff
@@ -26,57 +26,76 @@ module.exports = pa11y;
  */
 async function pa11y(url, options = {}, callback) {
 	const state = {};
-
-	/* eslint-disable prefer-rest-params */
-	// Check for presence of a callback function
-	if (typeof arguments[arguments.length - 1] === 'function') {
-		callback = arguments[arguments.length - 1];
-	} else {
-		callback = undefined;
-	}
-	/* eslint-enable prefer-rest-params */
+	let pa11yError = null;
```

Comment:
I had used `null` only because in the case of a successful test run, the callback was being given `null` as the first argument in the original code.  I see that tests pass when both variables are initialized as "undefined" and will revise.